browser/css/toolbar.css

```diff
@@ -1305,7 +1305,7 @@ button.leaflet-control-search-next
 [data-theme='dark'] .w2ui-icon.link{ background: url('images/dark/lc_inserthyperlink.svg') no-repeat center; }
 [data-theme='dark'] .w2ui-icon.insertsymbol{ background: url('images/dark/lc_insertsymbol.svg') no-repeat center; }
 [data-theme='dark'] .w2ui-icon.edit{ background: url('images/dark/lc_editdoc.svg') no-repeat center; }
-[data-theme='dark'] .w2ui-icon.fold{ background: url('images/dark/fold.svg') no-repeat center center / 19px; opacity: 0.8; filter: alpha(opacity=80);}
```

Comment:
isn't this change already present in https://github.com/CollaboraOnline/online/pull/9900 ? Shouldn't we just merge that first?

Reply:
@pedropintosilva oh i think i missed it let's first merge #9900 and then i rebase this